core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java

```diff
@@ -1465,4 +1465,12 @@
     String gitConsoleStagedFilesColor();
     String gitConsoleUnstagedFilesColor();
     String gitConsoleErrorColor();
+
```

Comment:
Looks here problem with styles, not good add SVN colors to the IDE API need find other solution. The same problem with GIT. @ashumilova what you think about it?